docs/web/diff.md

```diff
@@ -0,0 +1,903 @@
+  --------------------------------------------------
+  Checker name        | Severity | Number of reports
+  --------------------------------------------------
+  core.DivideZero     | HIGH     |                 1
+  deadcode.DeadStores | LOW      |                 1
+  --------------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that remained from first analysis to the next (via `--unresolved`): 
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --unresolved
+  ```
+  Expected output (no checkers are seen):
+  ```
+  ----======== Summary ========----
+  ---------------------------------------------
+  Number of processed analyzer result files | 0
+  Number of analyzer reports                | 0
+  ---------------------------------------------
+  ----=================----
+  ```
+
+## Review status rules and diffs
+
+Similarly to source code suppressions, you can set the review status of reports [on the GUI](../..//web/server/vue-cli/src/assets/userguide/userguide.md#review-status) as well, which will create a review status rule. Unlike source code suppressions, review status rules set the review status for _all reports_ matching that rule, _regardless_ whether the rule was created before or after the report was stored. Because these rules are stored on the server, **they play no part in local-local diffs as of yet**.
+
+:warning: Note: Review status rules act differently when runs are compared as opposed to when tags or dates are compared (see [below](#diff-on-tags-or-timestamps)).
+
+:warning: Note: If you are trying out with these examples, keep in mind that review status rules are _not_ deleted even if _all reports that it matches_ (by removing some or all runs in a given product) are. You will need to click on "Configuration" in the upper right corner, select "Review Status Rules", where these rules can be removed.
+
+### Example: Setup
+
+Create `example.cpp` the same way as the previous examples:
+
+```cpp
+// example.cpp
+
+int foo(int z)
+{
+  if (z == 0)
+    return 1 / z; // report by core.DivideZero
+
+  return 0;
+}
+
+int bar(int x)
+{
+  int y;
+  y = x % 2; // report by deadcode.DeadStores
+
+  return x % 2;
+}
+```
+Analyze and store it to a running CodeChecker server:
+```sh
+CodeChecker check -o ./test_report_dir -b "g++ example.cpp -c"
+CodeChecker store -n "Test Run" ./test_report_dir
+```
+
+Now, navigate to the the web GUI, and create review status rules. Click on "Test Run".
+
+![image](https://github.com/Szelethus/codechecker/assets/23276031/c308038d-466f-453c-ad41-e69cb789ff4d)
+
+Click on the first report, emitted by `deadcode.DeadStores`.
+
+![image](https://github.com/Szelethus/codechecker/assets/23276031/1e6c47d6-d9f6-44c9-a770-08108ed53a04)
+
+Set its review status to false positive.
+
+![image](https://github.com/Szelethus/codechecker/assets/23276031/d642a5aa-17f3-4566-856f-a54bd37b20d8)
```

Comment:
I think when this picture was taken, the brightness adjustment patch remained in the cache, just like the other pictures where you can see the error bubble.